jdbc-v2/src/main/java/com/clickhouse/jdbc/internal/JdbcConfiguration.java

```diff
@@ -55,6 +55,14 @@ public static boolean acceptsURL(String url) {
         return url.startsWith(PREFIX_CLICKHOUSE) || url.startsWith(PREFIX_CLICKHOUSE_SHORT);
     }
 
+    private String cleanUrl(String url) {
+        url = stripUrlPrefix(url);
+        if (url.startsWith("//")) {
+            url = "http:" + url;
```

Comment:
this is a tricky thing :-) 
because I would write `"jdbc:ch://cloudinstance.aws.com:8433"` expecting that client would guess by port the protocol